docs/source/references/api_spec.md

```diff
@@ -71,35 +71,57 @@ Hint and whether each object exists:
 
 **Protocol Upgrade**: `WebSocket`
 
-**Encoding**: JSON
+**Encoding**: [Parity Scale Codec](#parity-scale-codec)
 
 **Endpoint**: `/events`
 
-**Expects**: 
+**Expects**:
 
-First message after handshake from client: `SubscriptionRequest` [*](#iroha-structures)
+First message after handshake from client: `EventStreamSubscriptionRequest` [*](#iroha-structures)
 
-When server is ready to transmit events it sends: `SubscriptionAccepted` [*](#iroha-structures)
+When server is ready to transmit events it sends: `EventStreamSubscriptionAccepted` [*](#iroha-structures)
 
 Server sends `Event` and expects `EventReceived`  [*](#iroha-structures) after each, before sending the next event.
 
 **Notes**:
 
-Usually, the client  waits for Transaction events. 
+Usually, the client  waits for Transaction events.
 
 Transaction event statuses can be either `Validating`, `Committed` or `Rejected`.
 
-Transaction statuses proceed from `Validating` to either  `Committed` or `Rejected`. 
+Transaction statuses proceed from `Validating` to either  `Committed` or `Rejected`.
 However, due to the distributed nature of the network, some peers might receive events out of order (e.g. `Committed` before `Validating`).
 
 It's possible that some peers in the network are offline for the validation round. If the client connects to them while they are offline, the peers might not respond with the `Validating` status.
 But when the offline peers come back online they will synchronize the blocks. They are then guaranteed to respond with the `Committed` (or `Rejected`) status depending on the information found in the block.
 
+### Blocks stream
+
+**Protocol**: HTTP
+
+**Protocol Upgrade**: `WebSocket`
+
+**Encoding**: [Parity Scale Codec](#parity-scale-codec)
+
+**Endpoint**: `/block/stream`
+
+**Expects**:
+
+First message after handshake to initiate communication from client: `BlockStreamSubscriptionRequest` [*](#iroha-structures)
+
+When server is ready to transmit blocks it sends: `BlockStreamSubscriptionAccepted` [*](#iroha-structures)
+
+Server sends `Block` and expects `BlockReceived`  [*](#iroha-structures) after each, before sending the next block.
+
+**Notes**:
+
+This endpoint is used to subscribe to blocks stream and have blocks sent to the client as they are added to the blockchain
```

Comment:
I think it is important to note.

The endpoint also let's us specify the starting block. And will send all the old blocks after the starting height and till the latest committed block, only after that it will start streaming.

Reply:
done